Common Collections/Strings/Process Some Strings/task.md

```diff
@@ -0,0 +1,11 @@
+## Process Some Strings
+
+There's tons of useful standard library functions for strings. Let's try and use some of
```

Comment:
There are tons of useful standard library functions for strings. Let's try and use some of